test/mitmproxy/addons/test_proxyauth.py

```diff
@@ -109,6 +121,24 @@ def test_check():
         )
         assert not up.check(f)
 
+        ctx.configure(
```

Comment:
yeah actually, I looked a bit at solutions to install a small local ldap server and as I was saying it is a bit of an overkill for testing purposes but ldap3 integrates a way to mock an ldap server, I should look into that direction I guess